source/addComment.js

```diff
@@ -1,3 +1,23 @@
 $(document).ready(function(){
-  // add your code here
+  $('#new_comment_button').on('click', function () {
+    event.preventDefault()
+
+    $('#new_comment_button').hide()
+    $('<form id="new_comment" name="new_comment" style="display: block"></form>').appendTo("body")
```

Comment:
this entire section of building the form could be extracted out into a constant or function that does the building for you 👍